research/06-require-ana.tex

```diff
@@ -14,13 +14,13 @@ \subsection{Design goals}
 important guideline when defining and implement the invidual requirements. As
 such, they can be considered the bounds within which all requirements must fit.
 
-\paragraph{Language-agnostic} A REPL should be generated for each language
+\paragraph{Language-agnostic} The REPL should work with all languages
 defined with Spoofax. This means that no assumptions can be made about any
 language constructs or syntax.
 
-\paragraph{Autogeneration} The REPLs should be automatically generated, just
-like all of Spoofax's other services. This means that generating a shell should
-be integrated into Spoofax's build system and should not require any additional
+\paragraph{Autogeneration} The REPL should be automatically provided, just like
```

Comment:
The Spoofax build process should still autogenerate menu's for eclipse and other editors.